.vale/fixtures/AsciiDoc/ValidTableBlocks/testvalid.adoc

```diff
@@ -9,22 +9,42 @@
 |Cell 3, row 1
 |===
 
+Some text.
+
+//vale-fixture
 .Table two
 [options="header"]
 |====
 |Column Title|Column Title
 |content|content
+This content goes on and on.
 |====
 
+Some text.
 
+//vale-fixture
+[caption="test"]
```

Comment:
This testvalid.adoc isn't valid because the table attributes must contain either "cols=" or "options="